lib/site_template/_config.yml

```diff
@@ -29,6 +29,17 @@ markdown: kramdown
 theme: minima
 gems:
   - jekyll-feed
-exclude:
-  - Gemfile
-  - Gemfile.lock
+
+# Exclude from processing.
+#   The items below are excluded by default, to customize the list, uncomment the
+#   lines below and simply add to the list. Items removed will **not** be excluded
+#   from processing.
```

Comment:
This comment is a little confusing to me. I think what you're driving at is that the defaults go away if you create a custom `exclude` here. Let's try to be really really precise here.